tests/unit_tests/file_stream_test.py

```diff
@@ -1,72 +0,0 @@
-import itertools
```

Comment:
Where does this test live now?

Reply:
oh, maybe it is:
[test_file_stream_internal.py](https://github.com/wandb/wandb/pull/4066/files#diff-4c840927cbb094e7aef0c9ffb9513628130d37d5be8ab9a0735a452fc13cd70d)

Reply:
yep